clover_admin.rb

```diff
@@ -182,22 +182,64 @@ def self.object_action(...)
     form_options(wrapper: :div)
 
     link = lambda do |obj|
+      return "nil" if obj.nil?
```

Comment:
Up to you, but my recommendation would be:

```suggestion
      return "" unless obj
```